oxo.py

```diff
@@ -1,32 +1,188 @@
 class OxoBoard:
+    """
+    Game board class.
+
+    Attributes:
+        board (list of list of int): The gameboard.
+        player (int): The player who is currently playing.
+        winning_boards (list of list of int): The possible winning board states. Note that the numbers represent the
+                                              player's positions in the board as 0 - 8 (top to bottom, left to right).
+    """
+
+    board = []
+    player = 0
+
+    winning_boards = [
+                      # Horizontal wins
+                      [0, 1, 2],
+                      [3, 4, 5],
+                      [6, 7, 8],
+
+                      # Vertical wins
+                      [0, 3, 6],
+                      [1, 4, 7],
+                      [2, 5, 8],
+
+                      # Diagonal wins
+                      [0, 4, 8],
+                      [6, 4, 2]
```

Comment:
Hard-coding these is prone to errors and difficult to expand to different board sizes. Could they have beenm initialised programmatically?